sources/postgres/data.go

```diff
@@ -158,11 +158,14 @@ func convBytes(val string) ([]byte, error) {
 	return b, err
 }
 
-func convDate(val string) (civil.Date, error) {
+func convDate(conv *internal.Conv, val string) (interface{}, error) {
 	d, err := civil.ParseDate(val)
 	if err != nil {
 		return d, fmt.Errorf("can't convert to date: %w", err)
 	}
+	if conv.TargetDb == "experimental_postgres" {
```

Comment:
Use TARGET_EXPERIMENTAL_POSTGRES constant?

Reply:
We couldn't use the constant before because of circular dependency. Have now moved all constants to a top-level common package and now have updated to use constant as well.